www/index.js

```diff
@@ -110,6 +110,9 @@ examplesContainer.addEventListener("click", (event) => {
 nuinput.addEventListener("keydown", (event) => {
   if (event.key == "Enter") {
     if (event.metaKey || event.ctrlKey) {
+      event.preventDefault();
+      runCommand();
```

Comment:
Why did you change this? I reverted the behaviour on purpose, as the help text ("Press Ctrl+Enter (or Command+Enter on OSX) to add a new line (add linebreak)") indicates.

Reply:
I just got used to being able to simply hit enter to run a command 🙈 No I have to unlearn that again

Reply:
Oooooh. I misread the logic. So you wanted ctrl+enter to add a newline not to run the command? Sorry about that. I had thought you wanted ctrl+enter to run the command so I switched it. We can switch it back.